CONTRIBUTING.md

```diff
@@ -65,32 +65,6 @@ pytest tests/ --cov=src/
 It is not uncommon to have path issues when running pytest from the command line.
 Ensure that your $PATH variable contains the directory with the citrine-python repo.
 
-#### Docker
```

Comment:
As far as I can tell, no one is using this, given that the corresponding files haven't been changed in 5 years.